src/content/docs/ko/_nav.yaml

```
@@ -279,7 +285,10 @@
       items:
         - /ko/console/guide/payments
         - /ko/console/guide/billing-payments
-    - /ko/console/guide/integration
+    - slug: /ko/console/guide/integration
+      items:
+        - /ko/console/guide/channel-manage
+        - /ko/console/guide/smartrouting
```

이거 V1/V2 상관없이 표시되는 거 의도되신 부분이실까요?
아니시라면 아마 이렇게 수정하심 될 것 같아요 (안 될 수도 있어서 혹시 안 되시면 알려주셔요)
```suggestion
        - slug: /ko/console/guide/smartrouting
          systemVersion: v2
```

---

@XiNiHa 아 이거 콘솔에 있는 메뉴라 V1 가이드에서 안보이면 아예 없다고 생각할 거 같아서 메뉴는 보여주되 내용에 V2만 지원한다고 안내하려고 했습니다.

---

아하.... 그러면 원래대로 둬야겠네요